esp/src/src-react/components/Files.tsx

```diff
@@ -351,8 +351,8 @@ export const Files: React.FunctionComponent<FilesProps> = ({
         header={<CommandBar items={buttons} farItems={copyButtons} />}
         main={
             <>
-                <SizeMe monitorHeight>{({ size }) =>
-                    <div style={{ position: "relative", width: "100%", height: "100%" }}>
+                <SizeMe>{({ size }) =>
+                    <div style={{ width: "100%", height: "100%" }}>
```

Comment:
removal of position: "relative" is different than every other changed component (where it was left)